onnx/version_converter/convert.h

```diff
@@ -754,6 +754,10 @@ class DefaultVersionConverter : public BaseVersionConverter {
           "Pow", OpSetID(14), OpSetID(15)));
       registerAdapter(make_unique<CompatibleAdapter>(
           "Shape", OpSetID(14), OpSetID(15)));
+
+      /******** 15 -> 16 ********/
+      registerAdapter(make_unique<CompatibleAdapter>(
```

Comment:
From what I understand in the discussion, this is not a compatible extension. Don't we have to use a non-default value for the new attribute to get the old behavior? If so, the adapter should add that attribute to the node.

Reply:
Sorry that I don't have the context here... So previously RoiAlign-10's behaviour is same as using output_half_pixel. instead of half_pixel (default for RoiAlign-16). Then yes please add a new adapter for it. Thanks @gramalingam for catching this!

Reply:
Added an adapter `RoiAlign_15_16`.